### PR TITLE
#164069264 Setup redux

### PR DESCRIPTION
**Pull request by walimike**
### Description
Create the appropriate folder/file structure for React/Redux workflow for app.
Returns message for react app.

### Tasks completed
- Create a store 
- Create folder structure for container, reducers and actions
- Add redux dependencies to package.json

image

[#164069264](https://www.pivotaltracker.com/story/show/164069264)

### CODE REVIEW

**Review by NLSanyu — APPROVED** [review]
LGTM

File: .babelrc [hunk 1]
```diff
@@ -1,6 +1,3 @@
 {
-    "presets": [
-        "@babel/preset-env",
-        "@babel/preset-react"
-    ]
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```
Comment (Stanley-Okwii) [inline .babelrc]: @walimike , You know you this marks makes readability hard. It was readable the way it was before you changed it.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1 @@
+{"extends": "airbnb"}
```
Comment (Stanley-Okwii) [inline .eslintrc]: @walimike , Add a new line here

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb"
+}
```
Comment (Stanley-Okwii) [inline .eslintrc.json]: @walimike , Add a new line

**Review by Stanley-Okwii — CHANGES_REQUESTED** [review]
Good work

**Review by nyabongoedgar — APPROVED** [review]

**Comment by akram256** [comment]
Hey @walimike Could you please rename the PR, it does not follow our naming conventions

File: src/components/App.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
```
Comment (huxaiphaer) [inline src/components/App.js]: Nice work, but the convention we are going to use is placing the **stateful** components e.g `App.js` component in the **containers** folder, and placing the **stateless** in the **components** folder. So, in this case, shift this component in the **container** folder and leave this (*component* folder) for stateless components. 

When you resolve all these issues let me know.

**Review by huxaiphaer — COMMENTED** [review]
LGTM

**Review by Stanley-Okwii — APPROVED** [review]

**Pull request merged**